microsite/data/plugins/jira-dashboard

```diff
@@ -0,0 +1,10 @@
+---
+title: Jira Dashboard
+author: AxisCommunications
+authorUrl: https://github.com/AxisCommunications
+category: Agile Planning
+description: Allows you to fetch and display Jira issues for your entity. You get quickly access to issue summaries to achieve better task visibility and more efficient project management. 
+documentation: https://github.com/AxisCommunications/backstage-plugins
+iconUrl: https://github.com/AxisCommunications/backstage-plugins/blob/main/plugins/jira-dashboard/media/jira-logo.png
```

Comment:
This looks to be an official Atlassian Jira logo, can you confirm we have permissions to use this?

Reply:
Looks like perhaps you can: https://www.atlassian.com/legal/trademark#:~:text=You%20may%20use%20the%20Atlassian,transposing%20Atlassian's%20logo%20with%20your

Reply:
@benjdlambert what's your thoughts?